### ISSUE TITLE
A11y_AzureDataStudio_Selectthedeploymentoption_ScreenReader:NVDA does not annouce the the selected imgae as selected.

### ISSUE THREAD

**Issue by V-sheman** [microsoft/azuredatastudio#7596]
GitHubTags:#SQL Azure Data Studio; #A11y_AzureDataStudio#A11yMAS; #A11yTCS;#WCAG1.3.1,#DesktopApp;#NVDA#Win32;
**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727ff9fe68421b004bf270cd509e4c5de165
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
Additional Details:
NVDA Version:2019.2

**Steps to Reproduce :**
1.Install /Launch the application Azure data studio.
2.Navigate to connections and click on more action button.
3.Select “Deploy SQL Server big data cluster” from more action drop down.
4.Click on “Deploy SQL Server big data cluster” .
5.Start NVDA and observe the announcement.
**Actual:**
When the focus is going on the selected image or the radio button the NVDA does not announce the selected word for the selected image.
****Expected:**
NVDA should announce the selected item as selected.
**User Impact:**
Screen reader User will not able to understand on which server they are connected to.
**Attachment for Reference:**
A11y_AsureDataStudio_Selectthedeploymentoption_ScreenReader.pptx
Does this issue occur when all extensions are disabled?: Yes

**Comment by V-sheman** [comment]
Verified the issue Version: 1.15.0-insider (user setup)
Commit: 56695be14ac0160e7a4d23b7af939b98fdaaea45
Date: 2020-01-30T05:35:45.707Z
VS Code: 1.42.0
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.18363
Issue is fixed hence closing the issue.
[7596 _fixed.pptx](https://github.com/microsoft/azuredatastudio/files/4132752/7596._fixed.pptx)

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-May2019;
